Readme.md

```diff
@@ -52,13 +52,34 @@ The sent reply would be the object:
 
 The plugin accepts an options object with the following properties:
 
-+ `bodyLimit`: the maximum amount of bytes to process
++ `bodyLimit`: The maximum amount of bytes to process
 before returning an error. If the limit is exceeded, a `500` error will be
 returned immediately. When set to `undefined` the limit will be set to whatever
 is configured on the parent Fastify instance. The default value is
 whatever is configured in
 [fastify](https://github.com/fastify/fastify/blob/master/docs/Factory.md#bodylimit)
  (`1048576` by default).
++ `parser`: The default parser used is the querystring.parse built-in.  You can change this default by passing a parser function e.g. `fastify.register(require('fastify-formbody'), { parser: str => myParser(str) })`
```

Comment:
Can you please add a link o `qs` and a test that uses it?

Reply:
i assumed this change would require a bump to major and added a section in the readme that includes guidance on upgrading. 

the test titled _plugin should allow providing custom parser as option_ tests providing a custom parser as well as bc behavior.